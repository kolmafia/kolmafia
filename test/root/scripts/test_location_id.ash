// snarfblat locations
print($location[Noob Cave].id);
print($location[The Dire Warren].to_int());

// non snarfblat locations
print($location[Summoning Chamber].id);
print($location[The Lower Chambers].to_int());