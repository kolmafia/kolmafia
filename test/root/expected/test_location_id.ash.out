240
92
-1
-1
